### DISCUSSION TITLE
BrowserChrome

### DISCUSSION THREAD

**Discussion by SilverChinchilla** [cmu-sei/GHOSTS#88] [Q&A] [answered]
I have gotten FireFox to stay open however Chrome only blimps upon executing ghosts on the client. I modeled it in the config file almost exactly like the FireFox portion of the TimeLine JSON file on the client but no luck. Any ideas? Thank you.

**Comment by sei-dupdyke** [comment]
Hmm, a couple of things to check:

1. What version of chrome? 
2. What version of chromedriver?
3. can i see the config for the chrome handler here please?

**Reply by SilverChinchilla** [reply]
version 81.0.4044.92
tried the wiki you guys have up here but couldn't have any luck, but the chrome config looks like the following (spacing is off just cause i didn't include all the websites):
 {
      "HandlerType": "BrowserChrome",
      "HandlerArgs": {
        "isheadless": "false",
        "blockimages": "false",
        "blockstyles": "false",
        "blockflash": "false",
        "blockscripts": "false",
        "stickiness": "65",
        "stickiness-depth-min": "3",
        "stickiness-depth-max": "15",
        "incognito": "true",
        "javascript-enable": "true",
        "visited-remember": "10"
      },
      "Initial": "about:blank",
      "UtcTimeOn": "00:00:00",
      "UtcTimeOff": "24:00:00",
      "Loop": "True",
      "TimeLineEvents": [
        {
          "Command": "random",
          "CommandArgs": [
            "list of websites here"
         {
              "Uri": "http://httpbin.org/post",
              "Category": "cat1",
              "Method": "POST",
              "Headers": {
                "1": "a",
                "2": "b"
              },
              "FormValues": {
                "1": "a",
                "2": "b"
              }
            },
            {
              "Uri": "http://httpbin.org/put",
              "Category": "cat1",
              "Method": "PUT",
              "Headers": {
                "1": "a",
                "2": "b"
              },
              "Body": "body"
            },
            {
              "Uri": "http://httpbin.org/delete",
              "Category": "cat1",
              "Method": "DELETE"
            }
          ],
          "DelayAfter": 300000,
          "DelayBefore": 0
        }

**Comment by sei-dupdyke** [comment]
Do you see anything in the logs/app.log file specific to chrome or chromedriver?

**Reply by SilverChinchilla** [reply]
Not that I could see.

**Comment by sei-dupdyke** [comment]
Try changing `about:blank` to `https://www.google.com` or something similar.

**Reply by SilverChinchilla** [reply]
I changed it to https://google.com and there was no change, would it have to be changed somewhere else other than the config file on the client?

**Comment by sei-dupdyke** [comment, marked as answer]
You noted Chrome version 81.0.4044.92, I believe that maps to this version of ChromeDriver (CD):

https://chromedriver.storage.googleapis.com/index.html?path=81.0.4044.138/

Can you verify which version of CD you have, and if its not v81, can you please re-try with this linked version?

I thought it might be something simple. I also thought that CD version mis-matches would show in the logs, but I'm not 100% sure it would show.

**Reply by SilverChinchilla** [reply]
I will give the link a try, is there a specific location on the client that it needs to be?

**Comment by sei-dupdyke** [comment]
Thanks. CD is located in the same directory as the GHOSTS executable.

**Reply by SilverChinchilla** [reply]
That seems to have done the trick! Thanks a bunch for the help! Super stoked

**Comment by sei-dupdyke** [comment]
Awesome 🤘 - sorry for all the back and forth, glad it is working.
